notebooks/03_Sentiment_Analysis_with_padl.ipynb

```diff
@@ -1,5 +1,54 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "e871ce00",
+   "metadata": {},
+   "source": [
+    "## Install `PADL` and `sentencepiece`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "53fbce98",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python -m pip install git+https://github.com/lf1-io/padl.git\n",
+    "\n",
+    "!pip install sentencepiece"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e41f55ea",
+   "metadata": {},
+   "source": [
+    "## Download `Sentiment140` dataset with 1.6 million tweets"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e8a8253c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Make data directory if it doesn't exist\n",
+    "!mkdir -p data\n",
+    "!wget -nc https://nyc3.digitaloceanspaces.com/ml-files-distro/v1/sentiment-analysis-is-bad/data/training.1600000.processed.noemoticon.csv.zip -P data\n",
+    "!unzip -n -d data data/training.1600000.processed.noemoticon.csv.zip"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e642e44f",
+   "metadata": {},
+   "source": [
+    "----"
+   ]
+  },
```

Comment:
@jasonkhadka Thanks! Could you also uncomment the block below for all of the pip installs?

Reply:
And you can remove the `torch`, and `sentencepiece` installs